README.md

```diff
@@ -157,6 +156,95 @@ Working in progress....
+
+**Body** is the central part of each page and is dynamic. Its layout and content will change to align with the specific purpose of the page. For instance, the body on the product details page will contain images and descriptions of products, while on the checkout page, it will display a form for payment information.
+
+**Footer**, like the header, is a static component displayed at the bottom of every page (except the admin dashboard page). It typically contains company information, a contact form, and direct links to the company's social media profiles, such as Facebook and Instagram.
+
+### Home page
+
+A homepage featuring a hero banner, new arrival products, and a recommended product list would be structured to both captivate and guide the user's shopping experience from the moment they arrive on the site.
+
+![Home page](./docs/images/wireframes/homepage.png)
+
+### Sign-in page
+
+The sign-in page is accessible to all visitors, allowing them to log into the website. Additionally, this page offers links that guide new users to the sign-up page and provides a redirect to the 'Forgot Password' page for users who need to recover their login credentials.
+
+![Sign in page](./docs/images/wireframes/sign-in-page.png)
+
+### Sign up page
+
+The Sign-up page enables new visitors to create an account and includes a link for existing users to return to the Sign-in page.
+
+![Sign up page](./docs/images/wireframes/sign-up-page.png)
+
+### Forgot password page
+
+Users will be directed to the Forgot Password page after clicking the 'Forgot Password' link. They are required to enter their email address and click the 'Send' button to receive a link for resetting their password.
+
+![Forgot Password page](./docs/images/wireframes/forgot-password-page.png)
+
+### Reset password page
+
+Upon clicking the reset password link received via email, users will be taken to a page where they can establish a new password.
+
+![Reset Password page](./docs/images/wireframes/set-new-password-page.png)
+
+### Search page
+
+The search bar enables users to refine their product search using keywords or by selecting from the categories in the drop-down menu. The resulting matches will be presented on the search results page.
+
+![Search page](./docs/images/wireframes/search-page.png)
+
+### Product details page
+
+Clicking on a product image or name will direct users to the Product Details page, where they can find more information about the product and add it to their cart if they wish to make a purchase.
+
+![Product Details page](./docs/images/wireframes/product-details-page.png)
+
+### Shopping cart page
+
+When users click the cart icon in the header, they will be taken to the Cart page. Here, they can view all the products they have added and can choose to proceed with their purchase by clicking the 'Proceed to checkout' button or continue shopping by clicking the 'Continue shopping' button.

```

Comment:
Can you change the below wordings:

-  `will be taken` to `will be prompted` again? It's more formal.
-  `cart icon **on** the header`

Can you change this sentence:

`Here, they can view all the products they have added and can choose to proceed with their purchase....` 

to 

`Here, users can view all the items that have been added to cart and they have options to proceed to payment by clicking.... or to continue shopping by clicking .....`

Reply:
I will keep `cart icon in the header` as it is. Please refer to the attached photo.[
![Uploading Screenshot 2023-11-28 at 9.49.56 pm.png…]()
](url)

Reply:
I also will keep the sentence 'Here, they can view all the products they have added and can choose to proceed with their purchase....' as it is.

![Screenshot 2023-11-28 at 9 56 12 pm](https://github.com/irene2mana-T3A2-2023/SakuraPantryDocs/assets/98362650/6204ef61-aa04-4381-903f-d209132554a0)

Reply:
sure, no worries, thank you.